packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -131,13 +132,14 @@ export class SelectPageObject {
     }
 
     public clickOption(index: number): void {
-        if (index >= this.selectElement.options.length) {
+        const visibleOptions = this.getVisibleOptions();
+        if (index >= visibleOptions.length) {
             throw new Error(
                 '"index" greater than number of current displayed options'
             );
         }
 
-        const option = this.selectElement.options[index]!;
+        const option = visibleOptions[index]!;
         option.scrollIntoView();
         option.click();
     }
```

Comment:
`clickOptionWithDisplayText(displayText: string)` below is using `clickOption()` but with an index into the entire options array, not the visible options array, so I think its behavior is now wrong.